assets/src/application/ui/input/SelectInput/SelectInput.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { SelectProps, SelectValue } from 'antd/lib/select';
+import { Select } from 'antd';
+
+import { BaseInput } from '../BaseInput';
+
+interface SelectInputProps extends SelectProps<SelectValue> {
+	label: string;
+}
+
+const SelectInput: React.FC<SelectInputProps> = React.memo(({ id, label, className, ...rest }) => {
+	const htmlId = id ? `ee-select-input-${id}` : null;
+	return (
+		<BaseInput label={label} id={htmlId} className={className}>
+			<Select style={{ width: 'auto' }} id={htmlId} size='middle' dropdownMatchSelectWidth={true} {...rest} />
```

Comment:
plz add `width: 'auto'` via a base css class like `ee-select-input`
we want to avoid inline styles for production code (is ok for prototypes)